__tests__/app.test.js

```diff
@@ -403,4 +403,72 @@ describe("GET /api/users",()=>{
 
 })
 
+describe("GET /api/articles (with queries",()=>{
+
+    test("200: /api/articles?sort_by=author (default order should be descending)",()=>{
+      return request(app)
+      .get("/api/articles?sort_by=author")
+      .expect(200)
+      .then(({body:{articles}})=>{
+        expect(articles).toBeSortedBy("author",{
+          descending: true
+        })
+      })
+    })
+
+    //Test failing 
+    //endpoint works for all columns except COMMENT COUNT(column added by making a join)
+    // test("200: /api/articles?sort_by=comment_count (default order should be descending)",()=>{
+    //   return request(app)
+    //   .get("/api/articles?sort_by=comment_count")
+    //   .expect(200)
+    //   .then(({body:{articles}})=>{
+    //     expect(articles).toBeSortedBy("comment_count",{
+    //       descending: true
+    //     })
+    //   })
+    // })
+
```

Comment:
I think the above is failing because comment_count is a string. You can have a look at either casting the count column (worth a google) to change it to be a integer. Or you can change this test to coerce the value, have a look at the jest-sorted docs :)